src/core/network.js

```diff
@@ -115,20 +133,17 @@ var NetworkManager = (function NetworkManagerClosure() {
         pendingRequest.expectedStatus = 200;
       }
 
-      if (args.onProgressiveData) {
-        // Some legacy browsers might throw an exception.
-        try {
-          xhr.responseType = 'moz-chunked-arraybuffer';
-        } catch(e) {}
-        if (xhr.responseType === 'moz-chunked-arraybuffer') {
-          pendingRequest.onProgressiveData = args.onProgressiveData;
-          pendingRequest.mozChunked = true;
-        } else {
-          xhr.responseType = 'arraybuffer';
-        }
+//#if (GENERIC || FIREFOX || MOZCENTRAL)
```

Comment:
Because the intention is to only match non-Firefox browsers, not specifically Chrome. For example, if we would ever add special support for IE, then the use of `!CHROME` would be inconvenient.